config-sample.yaml

```diff
@@ -101,6 +101,11 @@ rtc:
   # # Set to true to enable mDNS name candidate. This should be left disabled for most users.
   # # when enabled, it will impact performance since each PeerConnection will process the same mDNS message independently
   # use_mdns: true
+  # # Set to false to disable strict ACKs for peer connections where LiveKit is the dialing side,
+  # # ie. subscriber peer connections. This will prevent clients that do not ACK peer connections
```

Comment:
:nit: Replace `This will prevent` with `Disabling strict ACKs will prevent`. Sorry about the nit pick. When the value is set to `true` in the example and use of `this` confuses me (I look at that sample value of `true` and correlate to `this` referring to that `true` value). So, requesting change to make it explicit.

Reply:
Thanks for the suggestion, this is definitely more clear! Just updated

Reply:
Thank you @treyhakanson . Merging it now.